### ISSUE TITLE
Create a release v1.0

### ISSUE THREAD

**Issue by jchen9314** [UBC-MDS/DSCI_522_jchen9314_apchiodo#10]
Hi Tony, any other changes you wanna make on README.md? If not, we can create a release.

**Comment by apchiodo** [comment]
@jchen9314 I agree.  I am going to take a look right now and see if I can set up so it aligns more so with Tiffany's recommendations.

**Comment by apchiodo** [comment]
In all honesty.. I think we should revert back to what we previously had as a question.  Don't you think?

**Comment by jchen9314** [comment]
@apchiodo Yes, I think we might revert back to previous one.

**Comment by jchen9314** [comment]
@apchiodo I updated it on README.md.
